.circleci/config.yml

```diff
@@ -1,44 +1,100 @@
 defaults:
-  node_cache: &node_cache
-    key: v-{{ .Environment.CACHE_VERSION }}-api-javascript-{{ .Branch }}-{{ checksum "yarn.lock" }}
+  image: &image
+    docker:
+      - image: circleci/node:11.6.0
+  yarn_cache: &yarn_cache
+    key: yarn_cache-v{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ checksum "yarn.lock" }}
+  yarn_install: &yarn_install
+    name: Install
+    command: yarn
+  setup_git: &setup_git
+    name: Setup Git user
+    command: |
+      git config --global user.email "ops@groupbyinc.com"
+      git config --global user.name "GroupBy Ops"
+      git config --global push.default simple
+
+      cp .circleci/prepare-commit-msg .git/hooks/prepare-commit-msg
+      chmod +x .git/hooks/prepare-commit-msg
 
 version: 2
 
 jobs:
+  ### Stores the current build number in a file to use in unique cache keys and branch names for this workflow.
+  ### The build number is guaranteed to be unique per build per project and is suitable for use as an identifier.
+  generate_cache_key_file:
```

Comment:
I don't think we need this anymore. The compiled files are small enough that we can persist to the workspace instead of requiring a cache like in the StoreFront repo.